content/get-started/get-started.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Getting Started
+seo-title: Getting Started
+---
+
+Iron Fish is cryptocurrency that allows for safe, secure, and private transactions.
+
+This guide is intended for crypto novices to understand what Iron Fish is and how they can get started using it. If you want to skip directly to installing and running Iron Fish, you can [install a node](/use/get-started/installation) now.
```

Comment:
`For people new to crypto` instead of `crypto novice` ?

Reply:
changed